### DISCUSSION TITLE
will cypress migrate to node 14?

### DISCUSSION THREAD

**Discussion by migreyes30** [cypress-io/cypress#14919] [Questions and Help] [answered]
I am curious on when cypress will upgrade to a node 13 or higher?

**Comment by bahmutov** [comment, marked as answer]
We use the same Node version as Electron. So once Electron upgrades, we will upgrade too.

**Reply by migreyes30** [reply]
i see that Electron is using the node 14, 

https://github.com/electron/electron/blob/e46446e7e4b0b435cd3764cd0913d2fb5b52f98a/DEPS#L18-L19

am i correct?
